core/examples/tutorial/sub_menu/page_elements.md

```diff
@@ -0,0 +1,32 @@
+Page Elements
+=============
+# Usage
+
+{{ page_element(
+    name,
```

Comment:
I believe this needs to be escaped in order to compile.

Reply:
This was not intended to compile, just as a documentation of the `page_element` function. Maybe I should go about documenting this some other way?

Reply:
Yea, I think some other documentation would be better.